package.json

```diff
@@ -32,39 +32,39 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "@headlessui/react": "^2.1.0",
-    "@heroicons/react": "^2.1.4",
-    "@netlify/plugin-nextjs": "^5.3.3",
-    "@next/third-parties": "^14.2.4",
-    "@storyblok/react": "^3.0.10",
+    "@headlessui/react": "^2.1.8",
+    "@heroicons/react": "^2.1.5",
+    "@netlify/plugin-nextjs": "^5.7.2",
+    "@next/third-parties": "^14.2.13",
+    "@storyblok/react": "^3.0.14",
     "@typeform/embed-react": "^3.17.0",
     "cnbuilder": "^3.1.0",
     "cookies-next": "^4.2.1",
-    "framer-motion": "^11.2.11",
-    "next": "^14.2.4",
-    "react-loading-skeleton": "^3.4.0",
+    "framer-motion": "^11.7.0",
+    "next": "^14.2.13",
+    "react-loading-skeleton": "^3.5.0",
     "react-player": "^2.16.0",
     "storyblok-rich-text-react-renderer-ts": "^3.2.0",
     "usehooks-ts": "^3.1.0"
   },
   "peerDependencies": {
-    "react": "^18.2",
-    "react-dom": "^18.2"
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1"
   },
   "devDependencies": {
     "@tailwindcss/container-queries": "^0.1.1",
-    "@types/node": "^20.14.8",
-    "@types/react": "^18.3.3",
+    "@types/node": "^22.7.2",
+    "@types/react": "^18.3.9",
     "@types/react-dom": "^18.3.0",
-    "autoprefixer": "^10.4.19",
+    "autoprefixer": "^10.4.20",
     "decanter": "^7.3.0",
     "eslint": "^8.57.0",
```

Comment:
I didn't update eslint manually because we went with the next eslint default set up when we use the nextjs starter. Version 9 isn't compatible at the moment so let's wait.